config/environments/test.rb

```diff
@@ -44,4 +44,6 @@
       secret_access_key: ENV.fetch('AWS_S3_EXTERNAL_BUCKET_SECRET_ACCESS_KEY'),
       stub_responses: true
     }
+
+  config.hosts << "filestore-app"
```

Comment:
i think just call `config.hosts.clear` for dev and test like you did for the other ones

Reply:
Done!